### PR TITLE
add itako recipe

### PR DESCRIPTION
**Pull request by simpleoier**

### CODE REVIEW

File: egs/itako/svs1/local/data.sh [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+mkdir -p ${ITAKO}
+
+train_set=tr_no_dev
+train_dev=dev
+recog_set=eval
+
+if [ ${stage} -le 0 ] && [ ${stop_stage} -ge 0 ]; then
+    log "stage 0: Data Download"
+    # The ITAKO data should be downloaded from https://zunko.jp/kiridev/login.php
```
Comment (ftshijt) [inline egs/itako/svs1/local/data.sh:38]: Is this the right link?

File: egs/itako/svs1/run.sh [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+fs=24000
+fmin=80
+fmax=7600
+n_fft=2048
+n_shift=300
+win_length=1200
+
+
+score_feats_extract=frame_score_feats   # frame_score_feats | syllable_score_feats
+expdir=exp/rnn_test
```
Comment (ftshijt) [inline egs/itako/svs1/run.sh:18]:
```suggestion
```

File: egs/itako/svs1/local/dataset_split.py [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+# midfiles = list(find_files_by_extensions(data_root_url, exts=[".mid"]))
+# for path in midfiles:
+#     if path[-6:] == "13.mid":
+#         path13 = path
+#     if path[-6:] == "14.mid":
+#         path14 = path
+# path_temp = path14[-6:] + "00.mid"
+# shutil.move(path13, path_temp)
+# shutil.move(path14, path13)
+# shutil.move(path_temp, path14)
```
Comment (ftshijt) [inline egs/itako/svs1/local/dataset_split.py:129]:
```suggestion
```

File: egs/itako/svs1/local/data.sh (hunk 1 above)
Comment (simpleoier) [reply egs/itako/svs1/local/data.sh]: updated.

File: egs/itako/svs1/run.sh (hunk 2 above)
Comment (simpleoier) [reply egs/itako/svs1/run.sh]: done

File: egs/itako/svs1/local/dataset_split.py (hunk 3 above)
Comment (simpleoier) [reply egs/itako/svs1/local/dataset_split.py]: resolved.

**Pull request merged**